src/preferences/dialog/dlgprefinterface.cpp

```diff
@@ -1,25 +1,64 @@
-#include <QList>
+#include "preferences/dialog/dlgprefinterface.h"
+
 #include <QDir>
-#include <QToolTip>
 #include <QDoubleSpinBox>
-#include <QWidget>
+#include <QList>
 #include <QLocale>
-#include <QDesktopWidget>
+#include <QScreen>
+#include <QToolTip>
+#include <QWidget>
 
-#include "preferences/dialog/dlgprefinterface.h"
-#include "preferences/usersettings.h"
 #include "control/controlobject.h"
 #include "control/controlproxy.h"
-#include "skin/skinloader.h"
-#include "skin/legacyskinparser.h"
-#include "control/controlobject.h"
+#include "defs_urls.h"
 #include "mixxx.h"
+#include "preferences/usersettings.h"
+#include "skin/legacyskinparser.h"
+#include "skin/skinloader.h"
 #include "util/screensaver.h"
-#include "defs_urls.h"
-#include "util/autohidpi.h"
+#include "util/widgethelper.h"
 
 using mixxx::skin::SkinManifest;
 
+namespace {
+
+const QRegExp kMinSizeRegExp("<MinimumSize>(\\d+), *(\\d+)<");
+
+//Returns TRUE if skin fits to screen resolution, FALSE otherwise
+bool checkSkinResolution(
```

Comment:
```suggestion
bool skinFitsScreenSize(
```

The comment becomes obsolete.